crates/burn-tensor/src/tests/ops/squeeze.rs

```diff
@@ -35,6 +35,58 @@ mod tests {
         let squeezed_tensor: Tensor<TestBackend, 3> = tensor.squeeze(2);
     }
 
+    /// Test if the function works with an empty slice
+    #[test]
+    fn should_squeeze_dims_with_empty_slice() {
+        let tensor = Tensor::<TestBackend, 3>::ones(Shape::new([1, 1, 3]), &Default::default());
+        let squeezed_tensor: Tensor<TestBackend, 1> = tensor.squeeze_dims(&[]);
+        let expected_shape = Shape::new([3]);
+        assert_eq!(squeezed_tensor.shape(), expected_shape);
+    }
+
+    /// Test if the function works with positive indices
+    #[test]
+    fn should_squeeze_dims_with_positive_indices() {
+        let tensor = Tensor::<TestBackend, 4>::ones(Shape::new([1, 3, 1, 5]), &Default::default());
+        let squeezed_tensor: Tensor<TestBackend, 2> = tensor.squeeze_dims(&[0, 2]);
+        let expected_shape = Shape::new([3, 5]);
+        assert_eq!(squeezed_tensor.shape(), expected_shape);
+    }
+
+    /// Test if the function works with negative indices
+    #[test]
+    fn should_squeeze_dims_with_negative_indices() {
+        let tensor = Tensor::<TestBackend, 4>::ones(Shape::new([2, 1, 3, 1]), &Default::default());
+        let squeezed_tensor: Tensor<TestBackend, 2> = tensor.squeeze_dims(&[-3, -1]);
+        let expected_shape = Shape::new([2, 3]);
+        assert_eq!(squeezed_tensor.shape(), expected_shape);
+    }
+
+    /// Test to make sure the function doesn't do anything if a non-singleton dimension is squeezed
+    #[test]
```

Comment:
Following my other comment, this test should panic

Reply:
Sounds good, I'll make the change.